.i18nrc.json

```diff
@@ -53,5 +53,8 @@
     "x-pack/plugins/infra/public/utils/loading_state/loading_result.ts",
     "x-pack/plugins/infra/server/graphql/types.ts",
     "x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts"
+  ],
+  "translations": [
```

Comment:
__note:__ we need to track these files somewhere for a compatibility checks and automatic fixes.